packages/braid-design-system/src/lib/components/Table/Table.docs.tsx

```diff
@@ -0,0 +1,1240 @@
+              {
+                column1: 'Semper',
+                column2: 'Interdum',
+                column3: 'Viverra',
+              },
+            ])}
+            <Table label="Table column widths example">
+              <TableHeader>
+                <TableRow>
+                  <TableHeadCell width="content">
+                    <Text>Status</Text>
+                  </TableHeadCell>
+                  <TableHeadCell width="30%">
+                    <Text>Lorem</Text>
+                  </TableHeadCell>
+                  <TableHeadCell>
+                    <Text>Ipsum</Text>
+                  </TableHeadCell>
+                  <TableHeadCell>
+                    <Text>Dolor</Text>
+                  </TableHeadCell>
+                  <TableHeadCell width="content" align="right">
+                    <Text>Actions</Text>
+                  </TableHeadCell>
+                </TableRow>
+              </TableHeader>
+              <TableBody>
+                {getState('rows').map((row: any) => (
+                  <TableRow key={row.column1}>
+                    <TableCell width="content">
+                      <Badge bleedY>Badge</Badge>
+                    </TableCell>
+                    <TableCell width="30%">
+                      <Text>{row.column1}</Text>
+                    </TableCell>
+                    <TableCell>
+                      <Text>{row.column2}</Text>
+                    </TableCell>
+                    <TableCell>
+                      <Text>{row.column3}</Text>
+                    </TableCell>
+                    <TableCell width="content" align="right">
+                      <ButtonIcon
+                        icon={<IconEdit />}
+                        label="Edit"
+                        size="small"
+                        variant="transparent"
+                        id={`edit-${row.column1}`}
+                      />
+                    </TableCell>
+                  </TableRow>
+                ))}
+              </TableBody>
+            </Table>
+          </>,
+        );
+
+        return {
+          code: code
+            .replaceAll(': any', '')
+            .replaceAll(' key={row.column1}', ''),
+          value,
+        };
+      },
+    },
+    {
+      description: (
+        <>
+          <Text>
+            Additionally, columns accept a <Strong>minWidth</Strong> and{' '}
+            <Strong>maxWidth</Strong> prop to further constrain their width.
+            Both props accept a number representing the number of pixels. As a
+            convenience when setting a <Strong>maxWidth</Strong>, all{' '}
+            <Strong>Text</Strong> components in the column will default{' '}
+            <TextLink href="/components/Text#limiting-the-number-of-lines">
+              maxLines
+            </TextLink>{' '}
+            to a single line, ensuring that content is truncated when its
+            exceeds the specified width.
+          </Text>
```

Comment:
```suggestion
          <Text>
            Additionally, columns accept a <Strong>minWidth</Strong> and{' '}
            <Strong>maxWidth</Strong> prop to further constrain their width.
            Both props accept a number representing the number of pixels.
          </Text>
          <Text />
          <Text>
            Setting these values is useful for columns when you want to allow
            content to truncate at smaller screen sizes.
          </Text>
          <Text>
            As a convenience when setting a <Strong>maxWidth</Strong>, all{' '}
            <Strong>Text</Strong> components in the column will default{' '}
            <TextLink href="/components/Text#limiting-the-number-of-lines">
              maxLines
            </TextLink>{' '}
            to a single line, ensuring that content is truncated when its
            exceeds the specified width.
          </Text>
```
Feel like this needs a bit more detail, but not sure what the best info is here. Just one suggestion